### DISCUSSION TITLE
Lost 2FA Please help

### DISCUSSION THREAD

**Discussion by christiangubana** [community/community#32662] [Other Feature Feedback, Questions, & Ideas]
@github-staff , I'm not sure if this is the right place to log this issue, but please guard me in the right direction, if possible.

I can't log into my GitHub account due to loss of 2FA, I'm however keen to provide any information you might need in order to regain access.
Thank you.

**Comment by airtower-luna** [comment]
This is not the right place to read GitHub staff, though some of them look at the forum now and then. Most of the people here (including myself) are "just" other users, and mentions out of the blue are considered impolite.

The documentation lists the options you have: [Recover an account with 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)

You can still try to actually [contact GitHub support](https://support.github.com/request) if none of those work, but I assume the chances aren't very good. Good luck!

**Reply by ghostinhershell** [reply]
Hey @christiangubana, if @airtower-luna's answer addressed your question here, do you mind marking that as the answer?
